src/test/java/reactor/netty/http/client/HttpClientTest.java

```diff
@@ -490,19 +549,21 @@ public void contentHeader() {
 
 		HttpResponseStatus r =
```

Comment:
I'm not sure what this test ascertains... but I made it check the `Content-Length` header in _both_ responses in the other PR.

Reply:
The idea here is to test that content will be send with GET request and the server will respond with 400 Bad request because it does not expect content with GET request.